### PR TITLE
Count no. of Set Bits

### PR DESCRIPTION
**Pull request by utkarsh006**
## Related Issue
- Count no. of Set Bits #1487

## Proposed Changes
- I have added the following things: -
- Problem Statement
- Examples
- Algorithm
- Code
- Time Complexity & Space Complexity

## Additional Info
[LINK OF THE PROBLEM STATEMENT](https://practice.geeksforgeeks.org/problems/set-bits0143/1)

## Checklist

- [x] ✅ My code follows the code style of this project.
- [x] 📝 My change requires a change to the README.
- [x] 🎀 I have updated the README accordingly.
- [x] 👀 I have read the [**CONTRIBUTING**](https://github.com/Lakhankumawat/LearnCPP/blob/main/Contributing.md) document.
- [x] ✨ I have added tests to cover my changes.
- [x] 🚩 All new and existing tests passed.
- [x] 🌟 ed the repo

We were supposed to find the no. of set bits in a given number.

I have added the Problem Statement, Examples, an Algorithm on how to approach this problem, a commented Code, and written the Time Complexity and Space Complexity. And finally updated the README.md.

This change helps to solve a very famous problem of Bit Manipulation i.e. Count no. of set bits and this problem helps in understanding the concept of Bit Manipulation.

I coded this problem in VS Code and tested the given problem on the test cases in the range: 1 ≤ N ≤ 10^9. And it is working fine for them.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Output Screenshots
| Screenshot #1      | Screenshot #2  |
| ----------- | ----------- |
|Input : N = 6|Input : N = 8|
|image|image|

### CODE REVIEW

File: B-BitManipulation/README.md [hunk 1]
```diff
@@ -33,4 +29,34 @@ Algorithm that uses bit masking to generate all possible subsets from a given ar
...
-- Problem must have small constraints as Bitmasking takes up to exponential time.
\ No newline at end of file
+- Problem must have small constraints as Bitmasking takes up to exponential time.
+
+
+# Count Set Bits
+
+## Problem Statement 
+Given a positive integer N, print count of set bits in it. 
+
```
Comment (Mrjoy832) [inline B-BitManipulation/README.md:39]: explain what is set bit ..for better understanding of visitors

File: B-BitManipulation/README.md [hunk 2]
```diff
@@ -33,4 +29,34 @@ Algorithm that uses bit masking to generate all possible subsets from a given ar
...
+             Output : 1
+             
+ ![image](https://user-images.githubusercontent.com/94545831/169095943-17f56cab-2bea-4ce6-bf8a-cbbfb5abfd9c.png)
+
+            
+## Algorithm
+- Given a number N.
+- Run the loop till number is not 0.
+- Do Bitwise AND operation of that number to its preceding number.
```
Comment (Mrjoy832) [inline B-BitManipulation/README.md:55]: Which loop are you talking of?
- explain the algo properly

**Review by Mrjoy832 — CHANGES_REQUESTED** [review]

**Comment by Mrjoy832** [comment]
# Added Readme
## LGTM🌟

**Comment by utkarsh006** [comment]
@Lakhankumawat Kindly review this pr. My previous Pr's were : 1300,1337,1352,1385,1408

**Review by Lakhankumawat — APPROVED** [review]
LGTM 🚩

Minor contribution , your Pr is not eligible for Level2 for such minor contributions also your contribution has a duplication raid .

**Pull request merged**